src/configuration.js

```diff
@@ -34,6 +34,7 @@ class Configuration {
       staticWebsiteConfiguration: Joi.object(),
       assetPath: Joi.string().default('dist'),
       assetMatch: Joi.string().default('**'),
+      ignoreMatch: Joi.string().default(''),
```

Comment:
This could also be done by allowing `assetMatch` to be an array (I just did it like that locally before I had seen this PR).
```suggestion
      assetMatch: [Joi.string().default('**'), Joi.array().items(Joi.string())],
```

@nicekiwi looks like even at the current package.json `globby` version of 8.0.1, the `globby.sync` method can accept an array of patterns. I tested the change I've suggested locally, and it works with the following to upload all but my map files. Any estimate on when something like this could be merged and released?
```js
pluginOptions: {
  s3Deploy: { 
    assetMatch: ['**', '!(**/*.map)'],
  }
}
```